src/features/journeys/store.ts

```diff
@@ -48,9 +48,12 @@ const journeysSlice = createSlice({
       if (instanceItem) {
         instanceItem.isStale = true;
         if (instanceItem.data) {
-          if (state.timelineUpdatesByInstanceId[instanceItem.data?.id]) {
-            state.timelineUpdatesByInstanceId[instanceItem.data?.id].isStale =
-              true;
+          const { id } = instanceItem.data;
```

Comment:
I'm not quite sure I understand what this id is. I think it should be identical to the `instanceId` that is passed in as the payload? If so, I'm not sure why it is being picked again from a more nested field here. I'd appreciate some pointers :)

Reply:
Yes, I agree that this should be the same as `instanceId`.